Indexed search enabled via site configuration (#502)
sourcegraph/sourcegraph: doc/admin/search.md

> # Search configuration

Reviewer: Add a link to this doc to `doc/admin/index.md` under the **Features:** list

## Indexing

> Indexed search allows Sourcegraph to quickly text search all of your repositories. However, it requires additional resource usage so is only enabled by default when deploying via Kubernetes. For the single docker image it requires enabling via the configuration [search.index](site_config/all.md#search-index-string-enum). If you have lots of code to index please ensure Sourcegraph is deployed to a well provisioned node.

Reviewer 2: ```suggestion
Sourcegraph can index the code in your repositories to speed up search; however, it requires additional resource usage so it is only enabled by default not running as a single docker image. You can enable indexed search when running as a single docker image by configuring [search.index](site_config/all.md#search-index-string-enum). If you have lots of code to index please ensure Sourcegraph is deployed to a well provisioned node.
```

> Sourcegraph can index the code in your repositories to speed up search; however, it requires additional resource usage so it is only enabled by default not running as a single docker image. You can enable indexed search when running as a single docker image by configuring [search.index](site_config/all.md#search-index-string-enum). If you have lots of code to index please ensure Sourcegraph is deployed to a well provisioned node.

Reviewer: put backticks around search.index
